test/unit/event/send-return-event.test.js

```diff
@@ -17,26 +17,26 @@ jest.mock('ffc-pay-event-publisher', () => {
   }
 })
 jest.mock('../../../app/processing/get-payment-request-by-invoice-frn')
-const getPaymentSchemeByInvoiceAndFrn = require('../../../app/processing/get-payment-request-by-invoice-frn')
+const { getPaymentRequestByInvoiceAndFrn } = require('../../../app/processing/get-payment-request-by-invoice-frn')
 jest.mock('../../../app/config')
-const config = require('../../../app/config')
+const { processingConfig, messageConfig } = require('../../../app/config')
 const { PAYMENT_SETTLEMENT_UNMATCHED, PAYMENT_SETTLED } = require('../../../app/constants/events')
 const { SOURCE } = require('../../../app/constants/source')
-const sendReturnEvent = require('../../../app/event/send-return-event')
+const { sendProcessingReturnEvent } = require('../../../app/event/send-return-event')
 
 let paymentRequest
 let settlement
 
 beforeEach(() => {
-  paymentRequest = JSON.parse(JSON.stringify(require('../../mocks/payment-request')))
+  paymentRequest = JSON.parse(JSON.stringify(require('../../mocks/payment-requests/payment-request')))
   settlement = JSON.parse(JSON.stringify(require('../../mocks/settlement')))
 
-  getPaymentSchemeByInvoiceAndFrn.mockResolvedValue(paymentRequest)
+  getPaymentRequestByInvoiceAndFrn.mockResolvedValue(paymentRequest)
 
-  config.useV1Events = true
-  config.useV2Events = true
-  config.eventTopic = 'v1-events'
-  config.eventsTopic = 'v2-events'
+  processingConfig.useV1Events = true
+  processingConfig.useV2Events = true
+  messageConfig.eventTopic = 'v1-events'
+  messageConfig.eventsTopic = 'v2-events'
 })
```

Comment:
Should the before and after each not be defined in the describe?

Reply:
There's two describes in this test and the `beforeEach` is outside both so it can be shared.  Otherwise we'd need to duplicate it in each describe